packages/ui/src/common/components/Stepper.stories.tsx

```diff
@@ -0,0 +1,54 @@
+import { Meta, Story } from '@storybook/react'
+import React from 'react'
+
+import { Modal, ModalHeader } from './Modal'
+import { Stepper, StepperProps } from './Stepper'
+import { StepDescriptionColumn, StepperBody, StepperModalBody, StepperModalWrapper } from './StepperModal'
+import { TemplateBlock } from './storybookParts/previewStyles'
+
+export default {
+  title: 'Common/Stepper',
+  component: Stepper,
+  argTypes: {
+    steps: {
+      control: false,
+    },
+  },
+} as Meta
+
+const Template: Story<StepperProps> = (args) => {
+  return (
+    <TemplateBlock>
+      <Modal onClose={() => undefined} modalSize="l">
+        <ModalHeader onClick={() => undefined} title="Modal with stepper" />
+        <StepperModalBody>
+          <StepperModalWrapper>
+            <Stepper {...args} />
+            <StepDescriptionColumn>Col 2</StepDescriptionColumn>
+            <StepperBody>Col 3</StepperBody>
+          </StepperModalWrapper>
+        </StepperModalBody>
+      </Modal>
+    </TemplateBlock>
+  )
+}
+
+export const Simple = Template.bind({})
+Simple.args = {
+  steps: [{ title: 'Stake' }, { title: 'Form' }, { title: 'Submit application' }],
+}
+
+export const Complex = Template.bind({})
+Complex.args = {
+  steps: [
+    { title: 'General parameters' },
+    { title: 'Working Group title & limits', isBabyStep: true },
+    { title: 'Starting date & duration', isBabyStep: true },
+    { title: 'Description' },
+    { title: 'Short & opening description', isBabyStep: true },
+    { title: 'Application process', isBabyStep: true },
+    { title: 'Reward' },
+    { title: 'Stake' },
+    { title: 'Application form' },
+  ],
```

Comment:
It would be cool to have a story with the active step in the middle, so we can see what past steps display like
```suggestion
  ],
  active: 3,
```

Reply:
That's a good point, but you can control the steps using "controls": 

![](https://user-images.githubusercontent.com/247363/115223980-ad0e3100-a10c-11eb-8b54-b1169e6af913.png)